bitnami/elasticsearch/templates/coordinating/serviceaccount.yaml

```
@@ -0,0 +1,22 @@
+{{- if and (include "elasticsearch.coordinating.enabled" .) .Values.coordinating.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "elasticsearch.coordinating.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

In order to allow the use of `namespaceOverride`: 
```suggestion
  namespace: {{ include "common.names.namespace" . | quote }}
```